analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -384,6 +394,11 @@ impl Instance {
         let _timer = local_metrics.flush_duration_histogram.start_timer();
 
         match policy {
+            TableFlushPolicy::Unknown => {
+                return Err(Error::UnknownPolicy {
+                    backtrace: Backtrace::generate(),
```

Comment:
I thought the backtrace will be generated automatically by snafu. Won't it?

Reply:
it would 👍